### ISSUE TITLE
Discovery on Product Roadmap Optimization tools - SF IdeaExchange

### ISSUE THREAD

**Issue by ehuthmacher** [openedx/wg-marketing#40]

**Comment by ehuthmacher** [comment]
Ehsan recommended for our community to use a tool such as Salesforce Community Cloud, which allows the community to put emphasis on items on the roadmap that should be prioritized, based on consumer demand and to help scale adoption for the platform. IdeaExchange is only used for Salesforce, so Cecilia will conduct discovery on similar options.

**Comment by marcotuts** [comment]
My thoughts on this - I would encourage us to build / expand our use of Github's Product board, or explore plugins in Discourse to allow for voting (example - https://github.com/discourse/discourse-post-voting). Expanding the audience that can easily provide input to the roadmap / direction of the platform is great, but ideally we could try to layer this into existing tools where possible.

**Comment by ehuthmacher** [comment]
Discussion: https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion

**Comment by marcotuts** [comment]
Another note from @sarina  - we also have product proposals in the confluence wiki, which would be helpful to link to, reorganize, or whatever is needed to help make them more visible to others. 

How all these tools / options connect together to support a wider community engaging on the platform direction is a tricky challenge for sure, thanks for taking this up.

**Comment by sarina** [comment]
Currently the product proposals are in https://openedx.atlassian.net/wiki/spaces/OEPM but they're not in what I think would be the most useful which is three folders, "Proposed", "Implementing", and "Implemented". It would require a reorganization of the wiki. I would want to follow up with @jmakowski1123 about this idea, but I could take it on if useful.

**Comment by ehuthmacher** [comment]
@jyliugithub I found the correct SF integration that Ehsan was referring to. It is [Ideas for Community](https://appexchange.salesforce.com/appxListingDetail?listingId=a0N3A00000EFoHiUAL). I will obtain pricing information for this integration for us.

**Comment by jmakowski1123** [comment]
> Currently the product proposals are in https://openedx.atlassian.net/wiki/spaces/OEPM but they're not in what I think would be the most useful which is three folders, "Proposed", "Implementing", and "Implemented". It would require a reorganization of the wiki. I would want to follow up with [@jmakowski1123](https://github.com/jmakowski1123) about this idea, but I could take it on if useful.

That is an easy tweak, just did it. (There are a few weird permissions issues to hammer out, but this makes sense as an organizational strategy moving forward.) https://openedx.atlassian.net/wiki/spaces/OEPM/pages/3901390856/Open+Source+Contributions+-+Product+Proposals

**Comment by ehuthmacher** [comment]
Yes, agreed @jmakowski1123. The great news is that the SF integration is free. I am now running it by Legal to ensure we are able to integrate the extension into our existing SF instance.

**Comment by jyliugithub** [comment]
> [@jyliugithub](https://github.com/jyliugithub) I found the correct SF integration that Ehsan was referring to. It is [Ideas for Community](https://appexchange.salesforce.com/appxListingDetail?listingId=a0N3A00000EFoHiUAL). I will obtain pricing information for this integration for us.

That's great since it is free. Thanks @jmakowski1123  to share what open edX has already. Let's see how Ideas for Community can be applied here. Then we can think of migrating current resources to one place and organize the way we want( access/categories...). Most important is that we need to make sure community members are aware of such tool. @ehuthmacher

**Comment by ehuthmacher** [comment]
@jyliugithub it looks like the [free tool in SF](https://appexchange.salesforce.com/appxListingDetail?listingId=a0N3A00000EFoHiUAL) has a lot of negative reviews. Could you kindly look into this in more detail and share an update at the next Marketing meeting? Perhaps, there is another (free) tool that may work better for voting purposes.

**Comment by jyliugithub** [comment]
> @jyliugithub it looks like the [free tool in SF](https://appexchange.salesforce.com/appxListingDetail?listingId=a0N3A00000EFoHiUAL) has a lot of negative reviews. Could you kindly look into this in more detail and share an update at the next Marketing meeting? Perhaps, there is another (free) tool that may work better for voting purposes.

Sure, will do.

**Comment by jyliugithub** [comment]
@ehuthmacher I have not found anything free, but there are some tools with free trial available , check here https://rapidr.io/blog/feature-voting-tools/   Among them, I like https://nolt.io/features ; https://rapidr.io ; and https://sleekplan.com I assume this requires a collective investigation from developer, product, business side. @sarina @jmakowski1123 Would you mind take a look at this too and give some advise ? Meanwhile,let me check with our own product team to see if they can recommend any free ones.

**Comment by jmakowski1123** [comment]
> @ehuthmacher I have not found anything free, but there are some tools with free trial available , check here https://rapidr.io/blog/feature-voting-tools/ Among them, I like https://nolt.io/features ; https://rapidr.io ; and https://sleekplan.com I assume this requires a collective investigation from developer, product, business side. @sarina @jmakowski1123 Would you mind take a look at this too and give some advise ? Meanwhile,let me check with our own product team to see if they can recommend any free ones.

Is there a very simple MVP approach we could use here, such as using the "thumbs up" icon in GH to vote?

**Comment by ehuthmacher** [comment]
Great, thank you @jmakowski1123 . FYI @jyliugithub

**Comment by jyliugithub** [comment]
> > @ehuthmacher I have not found anything free, but there are some tools with free trial available , check here https://rapidr.io/blog/feature-voting-tools/ Among them, I like https://nolt.io/features ; https://rapidr.io ; and https://sleekplan.com I assume this requires a collective investigation from developer, product, business side. @sarina @jmakowski1123 Would you mind take a look at this too and give some advise ? Meanwhile,let me check with our own product team to see if they can recommend any free ones.
> 
> Is there a very simple MVP approach we could use here, such as using the "thumbs up" icon in GH to vote?

@jmakowski1123 It depends on whom we want to give access to, if only product team and developers, yes, we can definitely take much simpler approach like Github, if we want to collectively consider feedbacks and suggestions from the whole community, even customers, we can look at other alternatives. @ehuthmacher The purpose of such tools is to collect community and customer feedbacks on product , roadmap, features, comments, as long as all the community have a public space to communicate on this purpose and they know where to access this is important, maybe we should not insist on a vote/unvote feature. We want it to be simple, easy to access, no learning curve for the tool itself. We can discuss on the MWG meeting together with product team again.

**Comment by cassiezamparini** [comment]
@ehuthmacher Thanks for linking me to this thread. I think this is a great idea if we can get it right! I've been sifting through the [CC Survey Data](https://discuss.openedx.org/t/survey-improving-the-core-contributor-experience/12409/2) and this kind of process could be greatly appreciated in further emphasising the sense of community - as in "your vote counts". I know I would appreciate it. 

> Currently the product proposals are in https://openedx.atlassian.net/wiki/spaces/OEPM but they're not in what I think would be the most useful which is three folders, "Proposed", "Implementing", and "Implemented".

- Thanks for implementing this change @jmakowski1123. This is helpful.-
- @marcotuts you mentioned linking to the wiki proposals - do you mean show the "votes" on each wiki proposal as well? If so, I 100% agree - even if it's a simple text section (for now) at the top of the each wiki proposal that indicates the "status of voting" / a link to the "votes"  (ie. Github issue). in other words some sort of transparency about what's actually happening, including the voting period which I know we're already trying to implement with Product Review periods. It could be up to the Product Coordinator to manage this. It's currently difficult to get a snapshot of where a Product Proposal is in the process without delving deeper into the content on the Wiki / Github. @sarina what do you think?

> Most important is that we need to make sure community members are aware of such tool. @ehuthmacher

I agree with @jyliugithub. I think sometimes community members feel a little out the loop when it comes to community processes.

**Comment by ehuthmacher** [comment]
@jyliugithub will present regarding this topic tomorrow.

**Comment by ehuthmacher** [comment]
@jyliugithub Proposal by Nadheem: https://www.discourse.org/plugins/topic-voting.html
